matchmaking.py

```diff
@@ -45,8 +45,8 @@ def choose_opponent(self):
         variant = self.matchmaking_cfg.get("challenge_variant") or "random"
         if variant == "random":
             variant = random.choice(self.variants)
-        base_time = self.matchmaking_cfg.get("challenge_initial_time", 60)
-        increment = self.matchmaking_cfg.get("challenge_increment", 2)
+        base_time = random.choice(self.matchmaking_cfg.get("challenge_initial_time") or [60])
+        increment = random.choice(self.matchmaking_cfg.get("challenge_increment") or [2])  # [0] evaluates to True

```

Comment:
I don't understand the comment on this line.

    # [0] evaluates to True

Is this an explanation for the choice of `[2]` as a default? A warning against putting only `- 0` in the user's `config.yml` file?

Reply:
No warning, just to make it clear that if the user puts 0 as the only option for the increment, it will not default the increment to 2. It will only default to 2 if no increment values are given.